### PR TITLE
Example code: vm-with-two-nics

### PR DESCRIPTION
**Pull request by shayshahak**
deploy a VM with two Network Interface Cards on Azure. relevant for deploying a Virtual Appliance for FireWall, Proxy Machine, Router and such

### CODE REVIEW

File: examples/vm-two-nics/main.tf [hunk 1]
```diff
@@ -0,0 +1,151 @@
...
+    admin_username = "${var.admin_username}"
+    admin_password = "${var.admin_password}"
+  }
+
+  os_profile_linux_config {
+    disable_password_authentication = false
+  }
+}
+
+
```
Comment (metacpp) [inline examples/vm-two-nics/main.tf]: Please delete the last line with white space.

File: examples/vm-two-nics/variables.tf [hunk 2]
```diff
@@ -0,0 +1,15 @@
...
+  description = "The location where the Resources will be provisioned. This needs to be the same as where the Image exists."
+}
+
+variable "admin_username" {
+  description = "The username associated with the local administrator account on the Virtual Machine"
+}
+
+variable "admin_password" {
+  description = "The password associated with the local administrator account on the Virtual Machine"
+}
```
Comment (metacpp) [inline examples/vm-two-nics/variables.tf]: Add a newline.

Comment (shayshahak) [reply examples/vm-two-nics/variables.tf]: done

Comment (shayshahak) [reply examples/vm-two-nics/variables.tf]: done

File: examples/vm-two-nics/main.tf (hunk 1 above)
Comment (shayshahak) [reply examples/vm-two-nics/main.tf]: done

**Pull request merged**